src/main/java/com/internet/shop/controllers/product/AddProductController.java

```diff
@@ -0,0 +1,35 @@
+package com.internet.shop.controllers.product;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/product/add")
```

Comment:
```suggestion
@WebServlet("/products/add")
```
keep your naming consistent